packages/react-native-sdk/src/utils/setClientDetails.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  setSdkInfo,
+  setDeviceInfo,
+  setOSInfo,
+  SfuModels,
+} from '@stream-io/video-client';
+import { getDeviceInfoLib } from './device-info/libs';
+import { Platform } from 'react-native';
+
+export const setClientDetails = () => {
+  // TODO: set valid version
+  setSdkInfo({
+    type: SfuModels.SdkType.REACT_NATIVE,
+    major: '0',
+    minor: '0',
+    patch: '0',
+  });
+
+  const deviceInfo = getDeviceInfoLib();
+
+  setOSInfo({
+    name: Platform.OS,
+    version: Platform.Version.toString(),
```

Comment:
`Platform.Version` is android specific.. `DeviceInfo.getSystemVersion()` seems better to me

Reply:
It seems to be working for iOS, and from what I read here, I couldn't find anything mentioned about its availability only for android. Please correct me if I am wrong.

https://reactnative.dev/docs/platform-specific-code#detecting-the-ios-version

Reply:
interesting 

according to https://reactnative.dev/docs/platform

this is android specific

and for iOS it is osVersion

Reply:
Really interesting - https://reactnative.dev/docs/platform#version

Reply:
@khushal87 Here is my take on this.. DeviceInfo.getSystemVersion() is much better

DeviceInfo.getSystemVersion()  output is uniform as it gives OS version, like below 

```
// iOS: "11.0"
// Android: "7.1.1"
```

Platform.Version gives API version for android and OS version for iOS

```
// iOS: "11.0"
// Android: 25
```

Reply:
Umm, in that case, it makes sense to have `DeviceInfo.getSystemVersion()`. Thanks for sharing the info. 😄